CHANGELOG.md

```diff
@@ -3,12 +3,22 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
+
+## [2.2.1]
 ### Changed
-* Updated dependencies
+* Updated dependencies ([#246])
+* Erroring test reading symbolic link to dir ([#229])
```

Comment:
And yeah, process-wise I think it'd be good to start encouraging contributors to update the changelog in their PRs.